tests/providers/snowflake/hooks/test_snowflake.py

```diff
@@ -102,6 +102,7 @@ def test_get_conn_params(self):
                                 'warehouse': 'af_wh',
                                 'region': 'af_region',
                                 'role': 'af_role'}
+        self.assertEqual(self.db_hook.snowflake_conn_id, 'snowflake_default')
```

Comment:
I _think_ that this is actually a change in the default, and `snowflake_conn_id` was the previous default.

Do you think it's worth changing this default? If so it needs a note in UPDATING.md

Reply:
Hmm yeah that could've been the intent. However, with the way it was implemented it, I believe it also would have overwrote an explicit `snowflake_conn_id` passed into the hook/operator on initialization. But good call, I think it does make sense to note this in `UPDATING.md`, added.